core/Cargo.toml

```diff
@@ -28,21 +29,23 @@ mc-core-types = { path = "./types" }
 mc-crypto-dalek = { path = "../crypto/dalek", default-features = false }
 mc-crypto-hashes = { path = "../crypto/hashes", default-features = false }
 mc-crypto-keys = { path = "../crypto/keys", default-features = false }
+mc-crypto-memo-mac = { path = "../crypto/memo-mac", default-features = false }
```

Comment:
Is this actually required? Enclaves are never going to use this...

Reply:
I suppose a similar question surrounds `AccountId` and `ShortAddressHash`...

Reply:
you're right the memo include from `core` can be dropped, that's a remnant from before i minimised this.

`ShortAddressHash` is used in the computation of a TxSummary so this is already in the enclave. previously it was created from a public address when streaming the summary, i needed to lift it so we can [stream](https://mobilecoinofficial.github.io/ledger-mob/ledger_mob_apdu/tx/struct.TxSummaryAddTxOutUnblinding.html) this in for hardware tx summary computation to avoid the need for full `AccountKeys` objects (and the account keys dependency).

`AccountId` is lifted from full-service because we needed a common way to refer to an account between full-service / desktop-wallet / the ledger CI. i'll move this to `mc-transaction-signer` which isn't included in `mc-transaction-core`.